gulpfile.js

```diff
@@ -101,8 +105,11 @@ gulp.task("dist-min", function(cb) {
   var minConfig = assign({}, distConfig);
   minConfig.output.filename = "react-slick.min.js";
   minConfig.plugins = minConfig.plugins.concat(
-    new webpack.optimize.UglifyJsPlugin({
-      compressor: {
+    new UglifyJsPlugin({
```

Comment:
@maddhruv I don't think we need to setup UglifyJsPlugin anymore.  webpack in production mode take care of all optimisations

Reply:
True! Kept that for some custom configs/options